migrations/20220213130743_/migration.sql

```diff
@@ -1,8 +0,0 @@
-/*
```

Comment:
This change is not related to the PR, please undo it

Reply:
Done.

Reply:
Actually this has not been done so I've done it on your branch

```
$ git rev-list -n 1 HEAD --  migrations/20220201085056_pulling_data/migration.sql
513e223ede1f8ddf9941dc9bda8c3be827111694

$ git checkout 513e223ede1f8ddf9941dc9bda8c3be827111694^ --  migrations/20220201085056_pulling_data/migration.sql migrations/20220213130743_/migration.sql
```

Reply:
OK.